a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -1302,21 +1302,21 @@ <h4>Display techniques for navigation support</h4>
 			</section>
 		</section>
 		<section id="charts-diagrams-and-formulas">
-			<h3 data-localization-id="charts-diagrams-formulas-title">Complex content</h3>
+			<h3 data-localization-id="charts-diagrams-formulas-title">Rich content</h3>
 
 			<div class="note">
 				<p>This key information can be hidden if metadata is
 					missing. Alternatively it can be stated that
 					<span
 						data-localization-id="charts-diagrams-formulas-unknown"
-						data-localization-mode="descriptive">accessibility of complex content not identified as being accessible</span>
+						data-localization-mode="descriptive">accessibility of rich content not identified as being accessible</span>
```

Comment:
"accessibility of rich content not identified as being accessible"

Shouldn't this text also be changed to match the change below? It should probably be: "rich content not identified as being accessible"